dojo/tools/api_msdefender/api_client.py

```diff
@@ -0,0 +1,84 @@
+import json
+import urllib.request
+import urllib.parse
+import requests
+
```

Comment:
Can you implement `test_connection`? `test_product_connection` would be nice as well but not important for your use cases.
You can find inspiration in multiple places
```bash
$ fgrep -r test_connection dojo/tools/api_*/api_client.py
dojo/tools/api_blackduck/api_client.py:    # def test_connection(self):
dojo/tools/api_bugcrowd/api_client.py:    def test_connection(self):
dojo/tools/api_cobalt/api_client.py:    def test_connection(self):
dojo/tools/api_sonarqube/api_client.py:    def test_connection(self):
```